setup.py

```diff
@@ -45,7 +45,7 @@ def package_data(pkg, roots):
 
 setup(
     name='xblock-poll',
-    version='1.6.3',
+    version='1.6.4',
```

Comment:
nit: could you bump this to 1.7 since it's adding a new feature?

Reply:
After bumping it you can squash the commits and merge it.

Reply:
Thanks @xitij2000. I've bumped to 1.7.0 and squashed and merged.